docs/introduction/di_in_python.rst

```diff
@@ -130,7 +130,7 @@ using :doc:`Dependency Injector <../index>`:
    :language: python
    :linenos:
 
-Userful links
+Useful links
 ~~~~~~~~~~~~~
```

Comment:
Can you, please, also align this line with previous one? Number of `~` should match the length of the header.